service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -173,12 +176,9 @@ void listDatasetsUsingAdminPermissions() {
     when(datarepoService.getDatasets()).thenReturn(datasets);
     when(rawlsService.getDatasets()).thenReturn(workspaces);
     when(samService.hasGlobalAction(SamAction.READ_ANY_METADATA)).thenReturn(true);
-    when(datasetDao.find(StorageSystem.TERRA_WORKSPACE, workspaces.keySet())).thenReturn(List.of());
-    when(datasetDao.find(StorageSystem.TERRA_DATA_REPO, datasets.keySet()))
-        .thenReturn(List.of(tdrDataset));
     when(datasetDao.listAllDatasets()).thenReturn(List.of(workspaceDataset, tdrDataset));
-    ObjectNode tdrJson = (ObjectNode) datasetService.listDatasets().getResult().get(0);
-    ObjectNode workspaceJson = (ObjectNode) datasetService.listDatasets().getResult().get(1);
+    ObjectNode tdrJson = (ObjectNode) datasetService.listDatasets().getResult().get(1);
```

Comment:
This order dependency is ugly, I'm looking at ways to fix this using the hamcrest json object matchers.